test/Feature/RpcSpec.hs

```diff
@@ -197,6 +197,16 @@ spec actualPgVersion =
           ]|]
           { matchHeaders = [matchContentTypeJson] }
 
+      when (actualPgVersion >= pgVersion110) $
+        it "can embed if rpc returns domain of table type" $ do
+          post "/rpc/getproject_domain?select=id,name,client:clients(id),tasks(id)"
+              [json| { "id": 1} |]
+            `shouldRespondWith`
+              [json|[{"id":1,"name":"Windows 7","client":{"id":1},"tasks":[{"id":1},{"id":2}]}]|]
+          get "/rpc/getproject_domain?id=1&select=id,name,client:clients(id),tasks(id)"
```

Comment:
These tests lead me to believe that it'd be possible to `GET /projects_domain`. But of course that doesn't work because `select * from projects_domain` also doesn't. I thought the DOMAIN could be used as an alias.

Reply:
I think the domain is just on top of the implicit composite type associated with every table. So it does not really have any connection to the table itself. What you could do with that, is e.g. adding additional constraints to the return value of a function returning a table type. But those constraints would not have any relevance for the actual data stored. Does that make sense?

Reply:
Yes, makes sense. I've not had this use case, but it's great that we can support it transparently.